src/style.css

```diff
@@ -1,3 +1,65 @@
+:root {
+  --blue: #00acdc;
+  --black: #000;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, serif;
+}
+
+ul {
+  margin-top: 0;
+  margin-bottom: 0;
+  padding-left: 0;
+  list-style-type: none;
+}
+
+img {
+  display: block;
+}
```

Comment:
You're styling the `img` tag directly here. It's better to style using classes. This way, you can avoid potential conflicts if you add another image to your webpage in the future. Remember the checklist item: 'Don't use tag names for styling (except html and body)'.